MSProject/LibraryTableView/TableCells/WBBookTableViewCell.swift

```diff
@@ -35,4 +35,31 @@ class WBBookTableViewCell: UITableViewCell {
 
     }
     
+    var bookCellViewModel: WBBookCellViewModel? {
```

Comment:
I made a really long comment before about these VMs, and I'm to lazy to copy and paste it here.